lib/kuzzle/kuzzle.ts

```diff
@@ -570,15 +572,14 @@ class Kuzzle extends KuzzleEventEmitter {
   private async _waitForImportToFinish () {
     const importTypes = Object.keys(this.importTypes);
 
-    while (importTypes.length) {
+    importTypes.forEach(async importType => {
```

Comment:
We try to avoid `forEach` loop because they are not respecting the synchronicity of the `await` keyword